generator/src/main/java/com/scottlogic/deg/generator/restrictions/BlacklistRestrictions.java

```diff
@@ -22,4 +23,9 @@ public boolean match(Object o) {
     public boolean isInstanceOf(Object o) {
         return true;
     }
+
+    @Override
+    public String toString() {
+        return String.format("NOT %s", Objects.toString(blacklist));
```

Comment:
Assuming `Objects.toString(blacklist)` is used over `blacklist.toString()` for null safety, this implies that either the above `matches(Object o)` method should null check.